code42.json

```diff
@@ -629,6 +629,242 @@
                 "type": "table"
             },
             "versions": "EQ(*)"
+        },
+                {
+            "action": "create user",
+            "identifier": "create_user",
+            "description": "Create a new Code42 user account",
```

Comment:
Do you think it's worth including a `"verbose":` mentioning that once a user is created, it may take a few minutes before you are able to add them to detection lists?

Tony did say there were use cases around that

Reply:
yeah, makes sense